src/main/scala/com/fulcrumgenomics/fastq/FastqToBam.scala

```diff
@@ -96,8 +96,9 @@ class FastqToBam
   @arg(          doc="Read group ID to use in the file header.")                               val readGroupId: String = "A",
   @arg(          doc="The name of the sequenced sample.")                                      val sample: String,
   @arg(          doc="The name/ID of the sequenced library.")                                  val library: String,
+  @arg(          doc="Library or Sample barcode sequence.")                                    val barcode: Option[String] = None,
   @arg(          doc="Sequencing Platform.")                                                   val platform: String = "illumina",
-  @arg(doc="Platform unit (e.g. '<flowcell-barcode>.<lane>.<sample-barcode>')")                val platformUnit: Option[String] = None,
+  @arg(doc="Platform unit (e.g. '<flowcell-barcode>.<lane>')")                                 val platformUnit: Option[String] = None,
```

Comment:
I don’t like this change. It won’t be a unique identifier as required by the spec of we merge samples pooled across lanes or flowcells for example. There’s no text that forbids sample barcode as part of this unique identifier either. 

see:
![IMG_5792](https://github.com/fulcrumgenomics/fgbio/assets/846274/a32bfe03-ad23-40a9-ab0b-301109121307)

Reply:
I found [a post](https://gatk.broadinstitute.org/hc/en-us/articles/360035890671-Read-groups) re sample-barcode  in GATK. Seems strange to me - since "platform unit" sound to me like a physical thing. reverting the doc change.